image/golem_blender_app/golem_blender_app/commands/verify.py

```diff
@@ -58,7 +59,6 @@
         task_manager.update_subtask_status(
             subtask_id,
             SubtaskStatus.FAILURE)
-        # TODO: provide some extra info why verification failed
```

Comment:
i think this TODO is still valid

Reply:
True, there is still a need for the reason but it is not implemented in the verifier

Reply:
ok, so we want to remove it?

Reply:
I'll bring it back